kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/sink/CassandraSinkTask.scala

```diff
@@ -87,7 +81,16 @@ class CassandraSinkTask extends SinkTask with StrictLogging {
   override def put(records: util.Collection[SinkRecord]): Unit = {
     require(writer.nonEmpty, "Writer is not set!")
     writer.foreach(w => w.write(records.toVector))
-    records.foreach(r => counter.put(r.topic() , counter.getOrElse(r.topic(), 0L) + 1L))
+    records.foreach(r => counter.put(r.topic(), counter.getOrElse(r.topic(), 0L) + 1L))
+    if (timestamp > 0) {
+      val newTimestamp = System.currentTimeMillis()
+      if (scala.concurrent.duration.SECONDS.toSeconds(newTimestamp - timestamp) >= 60) {
+        logCounts()
+      }
+      timestamp = newTimestamp
```

Comment:
yes. we initialize with 0 and I wanted to avoid the comparison for the first time.